site/examples/working-with-the-opa-repl/index.md

```diff
@@ -0,0 +1,270 @@
+---
+sort_order: 1000
+nav_id: MAIN_EXAMPLES
+xmp_id: WORKING_WITH_THE_OPA_REPL
+layout: examples
+
+title: Working with the OPA REPL
+---
+
+{% contentfor header %}
+
+# Working with the OPA REPL
+
+REPLs are great for learning new languages and running quick experiments. You can take advantage of OPA's REPL to quickly run ad-hoc queries or prototype policies.
+
+{% endcontentfor %}
+
+{% contentfor body %}
+
+## OPA’s REPL
```

Comment:
Do we want to title this Goals to make it consistent with the Docker Authorization example?